app/src/main/res/xml/changelog_master.xml

```diff
@@ -10,6 +10,7 @@
 		<change>Addresses will be saved</change>
 		<change>Round icon for new Android versions</change>
 		<change>Added New York, Ghana, Costa Rica</change>
+		<change>Added Managua</change>
```

Comment:
Please add this to the list one line above, since this will only be shown to users when they upgrade from transportr 1 to 2.